core/common/src/MaterialProps.ts

```diff
@@ -62,6 +62,16 @@ export interface TextureMapProps {
   pattern_mapping?: TextureMapping.Mode;
   /** Weight at which to combine diffuse image and color; if undefined, defaults to 1.0 */
   pattern_weight?: number;
+  /** If true, override the mapping mode with constant LOD mapping for the normal map. */
+  pattern_useConstantLod?: boolean;
+  /** The number of times the texture is repeated if pattern_useConstantLod is true.  Increasing this will make the texture pattern appear smaller, decreasing it will make it larger. */
```

Comment:
Please document the defaults for these properties when omitted.

Reply:
done